examples/rl_gsm8k/deepseek_math_eval/eval_utils.py

```diff
@@ -299,8 +300,9 @@ def math_equal(
         if call_with_timeout(symbolic_equal_process, prediction, reference):
             return True
     else:
-        if symbolic_equal(prediction, reference):
-            return True
+        pass
```

Comment:
it would be nice to add the reason as to why we've comment this out, which as far as I remember is that this code hangs